src/pages/register-a-beacon/primary-beacon-use.tsx

```diff
@@ -146,7 +146,8 @@ const PrimaryBeaconUse: FunctionComponent<FormPageProps> = ({
                       }
                     >
                       <Input
-                        id="otherPleasureVesselText"
+                        id="other-pleasure-vessel-text"
```

Comment:
Scratching my head on this one as well.  I think the headache of having to make sure `id` / `name` always matches what's in the cache is made because we pre-define what can go into the cache.  So the same data field is defined in two places: here in the form, but also in the `Vessel` model.  But that's a necessary evil, isn't it? Because you don't want to save anything to the cache unless you know what it is (e.g. to prevent injection attacks etc.)?  (Or is there a safe way of not pre-defining what can go into the cache?)

When we come to post to the API we'll have a third (!) place to manage naming consistency of these fields!  I don't know what the answer is, other than we keep talking about it as we are now 🤣